ddtrace/internal/telemetry/__init__.py

```diff
@@ -14,3 +16,34 @@
 telemetry_lifecycle_writer = TelemetryWriter()
 
 __all__ = ["telemetry_lifecycle_writer"]
+
+
+_ORIGINAL_EXCEPTHOOK = sys.excepthook
+
+
+def _excepthook(tp, value, traceback):
```

Comment:
Do we like this in telemetry more than in tracer? Right now it's only used for telemetry, but I can imagine it being used in other places at some point? Probably fine to keep here for now.

Reply:
I'd prefer to keep this here. Instrumentation Telemetry is not tracer specific. It can be used in the profiler or the debugger.